adage/templates/base.html

```diff
@@ -0,0 +1 @@
+index.html
```

Comment:
Do we have to add this symlink? Can the location of base.html be a setting for tribe_client to make it more configurable?